### PR TITLE
Add an example with react-loadable

### PR DESCRIPTION
**Pull request by glennreyes**
As requested [here](https://github.com/jaredpalmer/razzle/pull/381#issuecomment-339676694)

Thanks again for this wonderful project!

#277

### CODE REVIEW

**Comment by jaredpalmer** [comment]
Can you not make it a submodule? I'm don't think it will work with create-razzle-app

**Comment by glennreyes** [comment]
Haha shoot my bad, will change

**Comment by glennreyes** [comment]
@jaredpalmer Wanna look over it?

File: examples/with-react-loadable/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "my-razzle-app",
```
Comment (jaredpalmer) [inline examples/with-react-loadable/package.json]: the npm pkg should be called `razzle-examples-with-react-loadable`

File: examples/with-react-loadable/razzle.config.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const { ReactLoadablePlugin } = require('react-loadable/webpack');
+
+module.exports = {
+  modify: (config, { target }) =>
+    target === 'web'
+      ? {
```
Comment (jaredpalmer) [inline examples/with-react-loadable/razzle.config.js]: nit: just use an if statement here, easier for ppl to read.

File: examples/with-react-loadable/src/server.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    ${assets.client.css
+      ? `<link rel="stylesheet" href="${assets.client.css}">`
+      : ''}
+  </head>
+  <body>
+    <div id="root">${markup}</div>
+    ${process.env.NODE_ENV === 'production'
+      ? `<script src="${assets.client.js}"></script>`
+      : `<script src="${assets.client.js}"></script>`}
```
Comment (jaredpalmer) [inline examples/with-react-loadable/src/server.js]: there should be a cross origin attribute on script so react-error-overlay works in dev

**Comment by glennreyes** [comment]
@jaredpalmer Thanks for the feedback, just applied all the requests. 👍

**Pull request merged**